vision_stack/buoy_detect/README.md

```diff
@@ -0,0 +1,24 @@
+BUOY_DETECT
+-----------
+This repository contains the codes that uses ML (Machine Learning) techniques to detect the buoy. This is a combination of 3 parts - 
+<br>
```

Comment:
@icyflame : Fixed it.

@pranaypratyush : Sent you a PR at https://github.com/pranaypratyush/kraken_3.0/pull/2